nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-actions.js

```diff
@@ -845,6 +845,116 @@
             }
         },
 
+        /**
+         * Enable all controller services in the specified ProcessGroup.
+         *
+         * @argument {selection} selection      The selection
+         * @argument {cb} callback              The function to call when request is processed
+         */
+         enableAllControllerServices: function (selection,cb) {
+            if (selection.empty()) {
+                // build the entity
+                var entity = {
+                    'id': nfCanvasUtils.getGroupId(),
+                    'state': 'ENABLED'
+                };
+                updateResource(config.urls.api + '/flow/process-groups/' + encodeURIComponent(nfCanvasUtils.getGroupId()) + '/controller-services', entity).done(updateProcessGroup);
+            } else {
+                var componentsToStart = selection.filter(function (d) {
+                    return nfCanvasUtils.isProcessGroup(d3.select(this));
+                });
+
+                // ensure there are some component to start
+                if (!componentsToStart.empty()) {
```

Comment:
Controller Services are not started/stopped. They are enabled/disabled. Can we update the names/comments to reflect this?

Reply:
Done